playbooks/packstack.yaml

```diff
@@ -287,32 +286,59 @@
         - name: Look for HTTPS 5000/v3 url
-          shell: mysql keystone -s -N -e 'select url from endpoint where url LIKE "http://%5000/v3"' | wc -l
+          shell: >
+            mysql keystone -s -N
+            -e 'select url from endpoint where url LIKE "http://%5000/v3"'
+            | wc -l
           register: https_5000_v3
           changed_when: False
         - name: Look for HTTPS 35357/v3 url
-          shell: mysql keystone -s -N -e 'select url from endpoint where url LIKE "http://%35357/v3"' | wc -l
+          shell: >
+            mysql keystone -s -N
+            -e 'select url from endpoint where url LIKE "http://%35357/v3"'
+            | wc -l
           register: https_35357_v3
           changed_when: False
         - name: Update 5000/v2.0 endpoints in Keystone database
-          shell: mysql keystone -e "update endpoint set url='https://{{ server_fqdn }}:5000/v2.0' where url='http://{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:5000/v2.0';"
+          shell: >
+            mysql keystone
+            -e "update endpoint set url='https://{{ server_fqdn }}:5000/v2.0'
+            where
+            url='http://{{hostvars[inventory_hostname]['ansible_default_ipv4']['address']}}:5000/v2.0';"
           when: https_5000_v20.stdout != '0'
         - name: Update 35357/v2.0 endpoints in Keystone database
-          shell: mysql keystone -e "update endpoint set url='https://{{ server_fqdn }}:35357/v2.0' where url='http://{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:35357/v2.0';"
+          shell: >
+            mysql keystone
+            -e "update endpoint set
+            url='https://{{ server_fqdn }}:35357/v2.0' where
+            url='http://{{hostvars[inventory_hostname]['ansible_default_ipv4']['address']}}:35357/v2.0';"
           when: https_35357_v20.stdout != '0'
         - name: Update 5000/v3 endpoints in Keystone database
-          shell: mysql keystone -e "update endpoint set url='https://{{ server_fqdn }}:5000/v3' where url='http://{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:5000/v3';"
+          shell: >
+            mysql keystone
+            -e "update endpoint set url='https://{{ server_fqdn }}:5000/v3' where
+            url='http://{{hostvars[inventory_hostname]['ansible_default_ipv4']['address']}}:5000/v3';"
           when: https_5000_v3.stdout != '0'
         - name: Update 35357/v3 endpoints in Keystone database
-          shell: mysql keystone -e "update endpoint set url='https://{{ server_fqdn }}:35357/v3' where url='http://{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:35357/v3';"
+          shell: >
+            mysql keystone
+            -e "update endpoint set
+            url='https://{{ server_fqdn }}:35357/v3' where
+            url='http://{{hostvars[inventory_hostname]['ansible_default_ipv4']['address']}}:35357/v3';"
           when: https_35357_v3.stdout != '0'
```

Comment:
try to use something  from http://docs.ansible.com/ansible/latest/modules/list_of_database_modules.html#mysql for this block